### ISSUE TITLE
Uncaught TypeError: Cannot read property 'startsWith' of undefined

### ISSUE THREAD

**Issue by arggasasao** [richiksc/atom-live-server-plus#9]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.35.1 x64
**Electron**: 2.0.18
**OS**: Mac OS X 10.14.3
**Thrown From**: [atom-live-server-plus](https://github.com/richiksc/atom-live-server-plus) package 3.0.0

### Stack Trace

Uncaught TypeError: Cannot read property 'startsWith' of undefined

```
At /Users/je10202/.atom/packages/atom-live-server-plus/lib/atom-live-server-plus.js:138

TypeError: Cannot read property 'startsWith' of undefined
    at Object.startServer (/packages/atom-live-server-plus/lib/atom-live-server-plus.js:138:59)
    at HTMLElement.atomLiveServerStartServer (/packages/atom-live-server-plus/lib/atom-live-server-plus.js:66:51)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:350017)
    at CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:348492)
    at AtomEnvironment.dispatchApplicationMenuCommand (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:1:694296)
    at EventEmitter.t (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:1:701680)
    at emitThree (events.js:136:13)
    at EventEmitter.emit (events.js:217:7)
```

### Commands

```
     -0:19.5.0 atom-live-server:start-server (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.33.4 
atom-clock 0.1.16 
atom-json-editor 0.5.1 
atom-live-server 2.3.0 
atom-live-server-plus 3.0.0 
Atom-SFTP-sync 2.0.1 
atom-tag-wrapper 0.2.0 
atom-wrap-in-tag 0.6.0 
base16-syntax 1.8.1 
busy-signal 2.0.1 
drag-relative-path 1.1.3 
emmet 2.4.3 
file-icons 2.1.31 
intentions 1.1.5 
jQuery 1.2.6 
language-vue 0.24.0 
linter 2.3.0 
linter-ui-default 1.7.1 
minimap 4.29.9 
minimap-titles 1.15.0 
platformio-ide-terminal 2.9.1 
remote-sync 4.1.8 
SFTP-deployment 1.0.2 
sftp-upload-on-change 0.1.0 
split-diff 1.6.0 
```

**Comment by richiksc** [comment]
Probably a null safety issue that I can fix. What did you have open as your active pane item at the time in atom? Did you have a file open? Did you have a project directory open in atom at the time? Was the file saved in the project directory?

I also recommend you uninstall the original `atom-live-server` as `atom-live-server-plus` is meant to replace, not supplement `atom-live-server`.

**Comment by richiksc** [comment]
I was able to reproduce the issue. It is when you have a new, unsaved/unnamed file open as your active text editor in Atom. A fix is coming.

**Comment by richiksc** [comment]
Fixed in `v3.0.1`.
